t.json

```diff
@@ -0,0 +1,930 @@
+{
+    "merkleRoot": "0x04ac73f7906e8bebeea35fcacd46f8039edaf005b01811490f2af22d2dd7239a",
+    "cycle": 2389,
+    "startBlock": "21245912",
```

Comment:
should this be included? maybe a more descriptive name

Reply:
This is just an artifact of one of the old trees that was generated, should be deleted